### DISCUSSION TITLE
Automerge enabled by default?

### DISCUSSION THREAD

**Discussion by dylanCz** [renovatebot/renovate#26600] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

v17.32.2

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### What is your question?

I seem to be seeing a strange issue where automerge is enabled by default on my projects in gitlab and I can't figure out why. 
[Here's](https://gitlab.com/renovate-test4) a group on my gitlab which replicates the issue.

The sample-terraform-module repo is just a mock dependency, and the main-project-2 repo calls to an older version of it to force PRs to open.
The renovate-runner repo includes the mend-renovate template hosted [here](https://gitlab.com/renovate-bot/renovate-runner) with a simple config.

If you check the open merge request against the main-project-2 repo, you'll see automerge is enabled. This only seems to have happened while upgrading to version 17 of the mend-renovate template, but I'm still confused as to where automerge is being enabled. 

If this is better suited for the mend-renovate gitlab repo, let me know and I will move this there.
Appreciate any help!
Dylan

### Logs (if relevant)

<details><summary>Logs</summary>

```
DEBUG: PR is configured for automerge (repository=renovate-test4/main-project-2, branch=renovate/gitlab.com-renovate-test4-sample-terraform-module-3.x)
...
DEBUG: Branch summary (repository=renovate-test4/main-project-2)
       "cacheModified": true,
       "baseBranches": [{"branchName": "main", "sha": "91ff56c965969fc09e7645412a83c3a70f3ab036"}],
       "branches": [
         {
           "automerge": true,
           "baseBranch": "main",
           "baseBranchSha": "91ff56c965969fc09e7645412a83c3a70f3ab036",
           "branchName": "renovate/gitlab.com-renovate-test4-sample-terraform-module-3.x",
           "branchSha": "52248623583a1befb323972141ec1fbf41ffeb18",
           "isModified": false,
           "isPristine": true
         }
       ],
       "defaultBranch": "main",
       "inactiveBranches": ["renovate/lock-file-maintenance"]
```

</details>

**Comment by dylanCz** [comment, marked as answer]
Nevermind, appeared to be extending the config file from the https://github.com/renovatebot/renovate which was causing issues, solved!

**Reply by viceice** [reply]
yes, where do you get that present? it should not be used by others because of the assignee configuration.
